goldens/tsec-exemption.json

```diff
@@ -0,0 +1,17 @@
+{
+  "ban-trustedtypes-createpolicy": [
+    "../src/material/icon/trusted-types.ts"
+  ],
+  "ban-element-innerhtml-assignments": [
+    "../src/material/icon/icon-registry.ts"
+  ],
+  "ban-element-setattribute": [
+    "../src/cdk/a11y/aria-describer/aria-reference.ts",
+    "../src/material-experimental/mdc-checkbox/checkbox.ts",
```

Comment:
Do you know if any of these exceptions should be eventually removed, or do they all contain valid reasons for this exception

Reply:
The `ban-trustedtypes-createpolicy` exception is expected. We won't be able to remove it until we have better support to create trusted types for SVG.

The `ban-element-innerhtml-assignments` exception is a false positive, because `src/material/icon/trusted-types.ts` defines its own `TrustedTypes` interface instead of pulling the type from the `@types/trusted-types` package. I'm not sure why it's coded that way, but technically it can be removed.

The "ban-element-setattribute" ones are tricky.  I don't see anything that raise immediate alarms, but some of those exceptions are exposing the "setAttribute" interface to users of the the custom elements, which *can* be abused to bypass other checks (depending on the type of the underlying elements). Those are probably hard to remove, since it will require breaking changes to the programming interface of those custom elements. That said, so far we haven't seen XSS caused by those in google3, so it might not be a big issue.